web/src/click_handlers.js

```diff
@@ -388,6 +407,46 @@ export function initialize() {
         $row.find(".markdown_preview").show();
     });
 
+    $("body").on("mouseenter", ".compose_control_button", () => {
+        const buttons = document.querySelectorAll(`
+            .compose_control_button[data-tooltip-template-id],
+            .compose_control_button[data-tippy-content],
+            .compose_control_button_container
+        `);
+
+        const tippyInstances = [...buttons].map((button) => {
+            const tippyContent = button.dataset.tippyContent;
+            const tooltipTemplateId = button.dataset.tooltipTemplateId;
+
+            if (tippyContent) {
```

Comment:
We already have a `get_tooltip_content` method, which does the same, let's use that instead.